projects/angular/src/button/button-group/_button-group.clarity.scss

```diff
@@ -26,41 +25,29 @@
 
       &:last-child {
         .dropdown-toggle {
-          border-top-right-radius: button-variables.$clr-btn-border-radius;
-          border-bottom-right-radius: button-variables.$clr-btn-border-radius;
+          border-top-right-radius: var(--clr-btn-border-radius);
+          border-bottom-right-radius: var(--clr-btn-border-radius);
         }
       }
     }
 
     .btn {
       margin: 0;
-      vertical-align: top;
       overflow: hidden;
 
       label {
         height: 100%;
       }
 
-      input[type='radio']:focus,
-      input[type='checkbox']:focus {
-        $outline-shadow: 0 0 variables.$clr_baselineRem_5px
-          var(--clr-btn-group-focus-outline, button-variables.$clr-btn-group-focus-outline);
-        $ie-outline-shadow: 0 0 variables.$clr_baselineRem_5px button-variables.$clr-btn-group-focus-outline;
+      // ATM after token migration (alpha or beta v17)
+      // remove when detailed radio/checkbox groups are implemented
```

Comment:
Can you file a separate JIRA ticket to track this removal?

Reply:
JIRA ticket was created CDE-1310.